### PR TITLE
Fix error reporter tabbing

### LINKED ISSUE #32307: Tab order between text fields on error reporter is not as expected
**Describe the bug**
During manual testing of the error reporter for 6.2 (#32280) it was discovered that the tab order on the error reporter is not as expected and leads to poor UX for the user.

**To Reproduce**

1. Start workbench and go to Help->About Workbench. Check "Report Usage Data" is checked
1. Run Segfault with `DryRun=False` to get the error reporter
1. Click the name field to put the cursor in that box and type any text
1. Press tab --> focus goes somewhere else but it's not clear where. 

**Expected behavior**

The most sensible thing would be for the cursor to move right and down the screen as is usual on forms so from Name it would go to Email.

**Platform/Version (please complete the following information):**
Tested on:
 - OS: CentOS
 - OS Version: 7
 - Mantid Version: 6.1 nightly on 2021-08-25

Most likely occurs everywhere

### PR DESCRIPTION
**Pull request by DavidFair**
**Description of work.**
- Fixes the tab order in the UI file
- Ensure Qt places the cursor into the text entry box, and not some random field (e.g. the email line) first.

**To test:**
Follow the reproduction instructions in #32307 
- Double check that the cursor starts in the free-form text entry box

Fixes #32307 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by RichardWaiteSTFC — DISMISSED** [review]
Nice fix, tabbing works as expected and cursor is put in the text box.

**Comment by RichardWaiteSTFC** [comment]
Ah just noticed there is a merge conflict - I'll approve again when this is fixed.

File: qt/python/mantidqt/dialogs/errorreports/report.py [hunk 1]
```diff
@@ -76,6 +76,9 @@ def __init__(self, parent=None, show_continue_terminate=False):
         # Dialog window to show more details of the crash to the user.
         self.details_dialog = MoreDetailsDialog(self)
 
+        # Set default focus to the editing box, rather then letting qt try and guess
+        self.input_free_text.setFocus()
```
Comment (martyngigg) [inline qt/python/mantidqt/dialogs/errorreports/report.py:80]: Thanks for fixing this. @DanielMurphy22 Can I just check that we didn't deliberately set the first field to have the cursor as the Name field for some reason?

Comment (DanielMurphy22) [reply qt/python/mantidqt/dialogs/errorreports/report.py:80]: No I don't see any reason why we'd want that!

**Review by martyngigg — APPROVED** [review]

**Pull request merged**